portfolio/src/main/java/com/google/sps/data/CommentUtil.java

```diff
@@ -37,8 +37,8 @@ public static void addComment(String userNickname, String text){
      * @param text The response's message.
      * @param parentId The id of the parent response or comment.
      */
-    public static void addResponse(String userNickname, String text, long parentId){
-        Response response = new Response(userNickname, text, parentId);
+    public static void addResponse(String userNickname, String text, long parentId, String imageUrl){
```

Comment:
Optional, but I think this would be super handy for your use case, where you are adding in new a bunch of new parameters. It could be good to eventually create something called a Builder, where you only need to set the fields that are actually filled by the user. For example, if we only have the text, we could do something like Comment.newBuilder().setText("text").build(). If there is text and imageUrl, you would do Comment.newBuilder().setText("text").setImageUrl("imageUrl").build().  Then you could just have generic addResponse() and addComment() methods that will only construct the Comment and or Response with the available fields. 

To create the Builder, you would need to make a nested Builder class. It would have each of the fields need for Comment. Define a static method called "newBuilder" in Comment that will create a new Comment.Builder class. Then you should have setters in your Builder class to set each of the fields for Comment (e.g. Builder setText(String text) { this.text = text; return this; }). Finally, you would need a "build" method to convert a Comment.Builder into a Comment. You could probably just have a single Comment constructor to handle this, since you would be using default values within the Comment.Builder class. 

P.S. This may be a bunch of upfront work, so not going to block this PR for this, but just wanted to show the idea for this, and feel free to implement that if you think it would be helpful.